### PR TITLE
Add in process client api tests

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description

Add in process client api tests

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: tests/integration_test/data/test_configs/standalone_job/client_api.yml [hunk 1]
```diff
@@ -146,3 +146,61 @@ tests:
...
+    setup:
+      - python -c "from torchvision.datasets import CIFAR10; CIFAR10(root='/tmp/nvflare/data', train=True, download=True)"
+    teardown:
+      - rm -rf /tmp/nvflare/data
+  - test_name: "run decorator-in-process"
+    event_sequence:
+      - "trigger":
+          "type": "server_log"
+          "data": "Server started"
+        "actions": [ "submit_job decorator" ]
```
Comment (SYangster) [inline tests/integration_test/data/test_configs/standalone_job/client_api.yml:173]: this should be `decorator_in_process`?

File: tests/integration_test/data/test_configs/standalone_job/client_api.yml [hunk 2]
```diff
@@ -146,3 +146,61 @@ tests:
...
+    setup:
+      - python -c "from torchvision.datasets import CIFAR10; CIFAR10(root='/tmp/nvflare/data', train=True, download=True)"
+    teardown:
+      - rm -rf /tmp/nvflare/data
+  - test_name: "run lightning-client-api-in-process"
+    event_sequence:
+      - "trigger":
+          "type": "server_log"
+          "data": "Server started"
+        "actions": [ "submit_job lightning" ]
```
Comment (SYangster) [inline tests/integration_test/data/test_configs/standalone_job/client_api.yml:192]: `lightning_in_process`

File: tests/integration_test/data/test_configs/standalone_job/client_api.yml (hunk 1 above)
Comment (YuanTingHsieh) [reply tests/integration_test/data/test_configs/standalone_job/client_api.yml:173]: yes good catch

**Review by SYangster — APPROVED** [review]

**Pull request merged**